nirum_wsgi.py

```diff
@@ -251,10 +264,10 @@ def dispatch_method(self, environ):
         )
 
     def route(self, environ, start_response):
-        """Route
+        """Route HTTP request to Nirum service method.
 
-        :param environ:
-        :param start_response:
+        :param environ: WSGI environment value.
+        :param start_response: The function can start HTTP response.
```

Comment:
As the meaning of `start_response` is quite clear in terms of WSGI, we can merely describe this as “a WSGI `start_response` callable” here.